examples/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':android-simple'
+include ':android-activity-graphs'
```

Comment:
I don't think we need to build the non-android examples with Gradle. This isn't a free-for-all, we don't need examples of every possible usecase.